migrations/statictypes/account_type_migration_test.go

```diff
@@ -1206,233 +1206,250 @@ func TestAccountTypeRehash(t *testing.T) {
 
```

Comment:
Best viewed with whitespace off. Refactors the "monolithic" test case which tested all types at once into individual subtests.

The new dictionary key conflict detection failed the test, because e.g. `AuthAccount.Keys` and `PublicAccount.Keys` now both map to `Account.Keys`, but the dictionary contained both old keys/types